scripts/setup.sh

```diff
@@ -6,6 +6,9 @@ rust_version=$(<rust-toolchain)
 # rustup
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain $rust_version
 
+# cargo hack CLI
```

Comment:
Probably also unnecessary

Reply:
Removed in [0cb0dc8](https://github.com/NomicFoundation/hardhat/pull/4380/commits/0cb0dc81d17d19c1f20b00fc0918d2451b727e20)